Description: adjusted config files to use correct directories
 adjusted config files to use correct directories by packagelayout etc.
Author: Andreas Wirooks <andreas.wirooks@1und1.de>

--- a/gitlab-shell/config.yml.example
+++ b/gitlab-shell/config.yml.example
@@ -15,10 +15,10 @@
 # Give the canonicalized absolute pathname,
 # REPOS_PATH MUST NOT CONTAIN ANY SYMLINK!!!
 # Check twice that none of the components is a symlink, including "/home".
-repos_path: "/home/git/repositories"
+repos_path: "/var/lib/gitlab/repositories"
 
 # File used as authorized_keys for gitlab user
-auth_file: "/home/git/.ssh/authorized_keys"
+auth_file: "/var/lib/gitlab/.ssh/authorized_keys"
 
 # Redis settings used for pushing commit notices to gitlab
 redis:
@@ -30,7 +30,7 @@
 
 # Log file.
 # Default is gitlab-shell.log in the root directory.
-# log_file: "/home/git/gitlab-shell/gitlab-shell.log"
+log_file: "/var/log/gitlab/gitlab-shell.log"
 
 # Log level. INFO by default
 log_level: INFO
--- a/config/gitlab.yml.example
+++ b/config/gitlab.yml.example
@@ -31,7 +31,7 @@
     # relative_url_root: /gitlab
 
     # Uncomment and customize if you can't use the default user to run GitLab (default: 'git')
-    # user: git
+    user: git
 
     ## Email settings
     # Email address used in the "From" field in mails sent by GitLab
@@ -52,9 +52,8 @@
     ##   COLOR  = 5
     # default_theme: 2 # default: 2
 
-
     ## Users management
-    # default: false - Account passwords are not sent via the email if signup is enabled. 
+    # default: false - Account passwords are not sent via the email if signup is enabled.
     # signup_enabled: true
 
     # Restrict setting visibility levels for non-admin users.
@@ -165,7 +164,7 @@
   # GitLab Satellites
   satellites:
     # Relative paths are relative to Rails.root (default: tmp/repo_satellites/)
-    path: /home/git/gitlab-satellites/
+    path: /var/lib/gitlab/gitlab-satellites/
 
   ## Backup settings
   backup:
@@ -174,11 +173,11 @@
 
   ## GitLab Shell settings
   gitlab_shell:
-    path: /home/git/gitlab-shell/
+    path: /var/lib/gitlab/gitlab-shell/
 
     # REPOS_PATH MUST NOT BE A SYMLINK!!!
-    repos_path: /home/git/repositories/
-    hooks_path: /home/git/gitlab-shell/hooks/
+    repos_path: /var/lib/gitlab/repositories/
+    hooks_path: /var/lib/gitlab/gitlab-shell/hooks/
 
     # Git over HTTP
     upload_pack: true
--- a/config/unicorn.rb.example
+++ b/config/unicorn.rb.example
@@ -32,24 +32,24 @@
 
 # Help ensure your application will always spawn in the symlinked
 # "current" directory that Capistrano sets up.
-working_directory "/home/git/gitlab" # available in 0.94.0+
+working_directory "/usr/share/gitlab" # available in 0.94.0+
 
 # listen on both a Unix domain socket and a TCP port,
 # we use a shorter backlog for quicker failover when busy
-listen "/home/git/gitlab/tmp/sockets/gitlab.socket", :backlog => 64
+listen "/var/run/gitlab/gitlab.socket", :backlog => 64
 listen "127.0.0.1:8080", :tcp_nopush => true
 
 # nuke workers after 30 seconds instead of 60 seconds (the default)
 timeout 30
 
 # feel free to point this anywhere accessible on the filesystem
-pid "/home/git/gitlab/tmp/pids/unicorn.pid"
+pid "/var/run/gitlab/unicorn.pid"
 
 # By default, the Unicorn logger will write to stderr.
 # Additionally, some applications/frameworks log to stderr or stdout,
 # so prevent them from going to /dev/null when daemonized here:
-stderr_path "/home/git/gitlab/log/unicorn.stderr.log"
-stdout_path "/home/git/gitlab/log/unicorn.stdout.log"
+stderr_path "/var/log/gitlab/unicorn.stderr.log"
+stdout_path "/var/log/gitlab/unicorn.stdout.log"
 
 # combine Ruby 2.0.0dev or REE with "preload_app true" for memory savings
 # http://rubyenterpriseedition.com/faq.html#adapt_apps_for_cow
--- a/config/database.yml.postgresql
+++ b/config/database.yml.postgresql
@@ -4,11 +4,11 @@
 production:
   adapter: postgresql
   encoding: unicode
-  database: gitlabhq_production
+  database: gitlab
   pool: 10
-  # username: git
-  # password:
-  # host: localhost
+  username: gitlab
+  password: 'gitlab'
+  host: localhost
   # port: 5432 
   # socket: /tmp/postgresql.sock
 
@@ -18,10 +18,11 @@
 development:
   adapter: postgresql
   encoding: unicode
-  database: gitlabhq_development
+  database: gitlab
   pool: 5
-  username: postgres
-  password:
+  username: gitlab
+  password: 'gitlab'
+  host: localhost
   # socket: /tmp/postgresql.sock
 
 #
@@ -30,10 +31,11 @@
 staging:
   adapter: postgresql
   encoding: unicode
-  database: gitlabhq_staging
+  database: gitlab
   pool: 5
-  username: postgres
-  password:
+  username: gitlab
+  password: 'gitlab'
+  host: localhost
   # socket: /tmp/postgresql.sock
 
 # Warning: The database defined as "test" will be erased and
@@ -42,8 +44,9 @@
 test: &test
   adapter: postgresql
   encoding: unicode
-  database: gitlabhq_test
+  database: gitlab
   pool: 5
-  username: postgres
-  password:
+  username: gitlab
+  password: 'gitlab'
+  host: localhost
   # socket: /tmp/postgresql.sock
--- a/config/database.yml.mysql
+++ b/config/database.yml.mysql
@@ -5,11 +5,11 @@
   adapter: mysql2
   encoding: utf8
   reconnect: false
-  database: gitlabhq_production
+  database: gitlab
   pool: 10
-  username: git
-  password: "secure password"
-  # host: localhost
+  username: gitlab
+  password: 'gitlab'
+  host: localhost
   # socket: /tmp/mysql.sock
 
 #
@@ -19,10 +19,11 @@
   adapter: mysql2
   encoding: utf8
   reconnect: false
-  database: gitlabhq_development
+  database: gitlab
   pool: 5
-  username: root
-  password: "secure password"
+  username: gitlab
+  password: 'gitlab'
+  host: localhost
   # socket: /tmp/mysql.sock
 
 # Warning: The database defined as "test" will be erased and
@@ -32,8 +33,9 @@
   adapter: mysql2
   encoding: utf8
   reconnect: false
-  database: gitlabhq_test
+  database: gitlab
   pool: 5
-  username: root
-  password:
+  username: gitlab
+  password: 'gitlab'
+  host: localhost
   # socket: /tmp/mysql.sock
--- a/config/resque.yml.example
+++ b/config/resque.yml.example
@@ -1,3 +1,3 @@
 development: redis://localhost:6379
 test: redis://localhost:6379
-production: redis://redis.example.com:6379
+production: redis://localhost:6379
--- a/lib/support/init.d/gitlab.default.example
+++ b/lib/support/init.d/gitlab.default.example
@@ -11,4 +11,4 @@
 
 # app_root defines the folder in which gitlab and it's components are installed.
 # The default is "/home/$app_user/gitlab"
-app_root="/home/$app_user/gitlab"
+app_root="/usr/share/gitlab"
